README.md

```diff
@@ -3,9 +3,8 @@
 ## ![Product Logo](./brand/brand-blue-48px.png) Accessibility Insights for Windows
 
 [![Build Status](https://dev.azure.com/accessibility-insights/Accessibility%20Insights/_apis/build/status/Microsoft.accessibility-insights-windows-CI?branchName=master)](https://dev.azure.com/accessibility-insights/Accessibility%20Insights/_build/latest?definitionId=3&branchName=master)
-
-
-
+[![Downloads](https://img.shields.io/github/downloads/microsoft/accessibility-insights-windows/total.svg)](https://github.com/Microsoft/accessibility-insights-windows/releases/latest)
+[![Release](https://img.shields.io/github/release/microsoft/accessibility-insights-windows.svg)](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0794.01)
```

Comment:
>v1.1.0794.01 [](start = 170, length = 12)

Is this something that requires manual updates when the release version changes? I'd love to have a way to reduce the risk of someone forgetting to update this when a new release goes out

Reply:
good point, I should change it to point to /releases/latest. It just means both badges will point to the same place.